microsoft-edge/devtools-guide-chromium/memory-inspector/memory-inspector-tool.md

```diff
@@ -21,48 +22,176 @@ ms.date: 02/15/2022
+
+1. Click **Toggle value type settings** (![Toggle value type settings blue](../media/memory-inspector-value-type-settings-toggle-back.png)) to close the value type settings.
+
+1. Use the drop-down menus to change the encoding from `dec` to `sci`. The value representations are updated.
+    
+    ![Memory Inspector address values updated view](../media/memory-inspector-address-values-updated-sci.png)
+    
+1. Explore the memory buffer using the keyboard or navigation bar.
+
+1. Repeat step **3** to observe value changes.
+
+
+<!-- ====================================================================== -->
+## WebAssembly memory inspection
 
-1. In the debugger, in the **Scope** section, expand **Local**.
+For WebAssembly (Wasm) memory inspection, the process is similar to inspecting JavaScript memory.
 
-   ![The Memory Inspector tool.](../media/memory-inspector-tool.png)
+1. Open the Wasm test site [http://memory-inspector.glitch.me/demo-wasm.html](http://memory-inspector.glitch.me/demo-wasm.html). <!-- To do item: copy to Microsoft Repo and reference there -->
 
-1. Expand the **buffer** to reveal the **Module Scope**.
+1. Open DevTools by pressing `F12` or `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).
 
-1. To the right of the **buffer** name, click the **Reveal in Memory Inspector panel** (!['Reveal in Memory Inspector panel' icon.](../media/reveal-in-memory-inspector-panel-icon.png)) icon.  Or, right-click the buffer, and then select **Reveal in Memory Inspector panel**.
+1. Click **Sources** and open the `memory-write.wasm` file.
 
-1. The **Memory Inspector** tool opens in the Drawer.  In the **Memory Inspector** tool, examine the **buffer**.
+1. Set a breakpoint at the first line in the loop, hexadecimal value **0x03c**.
 
-1. To inspect **Uint8Array b2**, <!-- expand that node to see the buffer, and then select the **Memory** icon (or-->right-click **b2**, and then select **Reveal in Memory Inspector panel**.
+1. Refresh the page.
 
-1. To inspect **Uint8Array b1**, <!-- expand that node to see the buffer, and then select the **Memory** icon (or-->right-click **b1**, and then select **Reveal in Memory Inspector panel**.  That re-focuses on the **Memory** tab of the first buffer.
+1. In the debugger pane under **Scope**, expand the **Module**.
 
-1. In the debugger, step, and see updates to buffers in the **Memory Inspector** tool.
+    ![Memory Inspector set breakpoint in Wasm file](../media/memory-inspector-wasm-breakpoint.png)
 
+1. Click on the **Reveal in Memory Inspector panel** icon at the end of the `$imports.memory` property line (![Reveal in Memory Inspector panel icon](../media/memory-inspector-open-from-buffer-icon.png)). 
 
+    The Wasm ArrayBuffer opens in Memory Inspector.
+    
+    ![Memory Inspector panel Wasm](../media/memory-inspector-panel-wasm.png)
 
 
 <!-- ====================================================================== -->
```

Comment:
I don't think the doc you've linked in the [!NOTE] is the right one. https://developer.chrome.com/blog/memory-inspector/ is a link to a blog titled "Introducing the Memory Inspector" which has good information about what this tool does but the actual doc we modified is: https://developer.chrome.com/docs/devtools/memory-inspector/

I would propose linking to the "Introducing the Memory Inspector" as an entry under a new "See also" heading or something like that

Reply:
fixed